packages/components/src/InputField/InputField.purs

```diff
@@ -101,6 +104,7 @@ render self@{ props, state } =
                 else mempty
             , disabled: props.disabled
             , autoComplete: if props.autoComplete == "" then unsafeCoerce Nullable.null else props.autoComplete
+            , autoFocus: props.autoFocus
```

Comment:
What are the exact semantics of setting this `false` instead of `null`? Initial Mosaico had a bunch of problems because it set all the Boolean fields as false and that didn't do what you'd expect with boolean attributes like `autoFocus`. Could you please check if this will end up making every input field have it enabled?

Reply:
Fine by me. This records stuff is by necessity a bit low level and some coercing to `null`s may be required.